android/engine/src/test/java/org/smartregister/fhircore/engine/task/FhirCarePlanGeneratorTest.kt

```diff
@@ -984,6 +984,91 @@ class FhirCarePlanGeneratorTest : RobolectricTest() {
       }
   }
 
+  @Test
+  @ExperimentalCoroutinesApi
+  fun `Generate CarePlan should generate child immunization schedule with pre-req def and expiry timing`() =
```

Comment:
Where we are making assertion for changes made into this PR like new properties?